classic-editor.php

```diff
@@ -326,7 +326,7 @@ public static function user_settings() {
 		?>
 		<table class="form-table">
 			<tr class="classic-editor-user-options">
-				<th scope="row"><?php _e( 'Editor', 'classic-editor' ); ?></th>
+				<th scope="row"><?php _e( 'Default editor for current user', 'classic-editor' ); ?></th>

```

Comment:
Don't think we need "for current user" here, it is implied as this is in the user's profile.

I'm 50/50 on whether "Default editor" is better than just "Editor".

Reply:
@azaozz -> Default Editor (also for me) is more explicative (understandable) of the action that only Editor. ;)